kuma/wiki/content.py

```diff
@@ -594,7 +594,7 @@ def process_header(self, token, buffer):
             start_inc = 2
             slug_base = slug
             while slug in self.known_ids:
-                slug = u'{0}_{1}'.format(slug_base, start_inc)
+                slug = u'%s_%s' % (slug_base, start_inc)
```

Comment:
I was grepping for `.format` and decided to change this anyway. Sorry for the churn.